tests/manage/z_cluster/nodes/test_rolling_terminate_and_recovery.py

```diff
@@ -74,17 +85,36 @@ def finalizer():
 
     def rolling_terminate_and_recovery_of_ocs_worker_nodes(self, nodes):
```

Comment:
Please add a docstring to the function. Please add an explanation about what happens with and without ms.

Reply:
Okay, that will do.